BuildingSync.xsd

```diff
@@ -5929,6 +5929,17 @@
           </xs:sequence>
         </xs:complexType>
       </xs:element>
+      <xs:element name="AuditFilingStatus" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>The status of audit filing.</xs:documentation>
+        </xs:annotation>
+        <xs:simpleType>
+          <xs:restriction base="xs:string">
+            <xs:enumeration value="Initial filing"/>
+            <xs:enumeration value="Amended Filing"/>
```

Comment:
Not sure what our convention is, (@nllong ?) but should probably be consistent with capitalization

Reply:
> Not sure what our convention is, (@nllong ?) but should probably be consistent with capitalization

Thanks for catching it! I think it should be "Amended filing". Correct me if I'm wrong @nllong .

Reply:
yes, `Amended filing` is the correct casing. For most cases we are using sentence case